src/metrics/extract_ttfs.py

```diff
@@ -90,9 +91,9 @@ def main():
             height = depth = italic = skew = width = 0
             glyph = glyf[name]
             if glyph.numberOfContours:
-                height = glyph.yMax
-                depth = -glyph.yMin
-                width = glyph.xMax - glyph.xMin
+                height = glyph.yMax / unitsPerEm
+                depth = -glyph.yMin / unitsPerEm
```

Comment:
During refactoring the generation scripts of fonts and their metrics, I discovered that `yMax` and `yMin` from generated `TTF` does not match `height` and `depth` from `tfm`(TeX font metrics).

Are their definitions different? Or, do they change during conversion process? If so, which metrics should KaTeX use?

CC @kevinbarabash

Reply:
I just did some spot checking of Typewriter-Regular (as that's what I was working on recently) and the letters I checked seemed close, but there's definitely some error. For example, `y` has a `yMin` of `-228` and `yMax` of `431`, and the metrics file says `[0.22222, 0.43056, ...]`.  That's more than floating-point rounding should account for... I'll turn this into an issue.

Reply:
@ylemkimon the `height` and `depth` calculations look reasonable. 

@edemaine the difference may be due to when we're getting the measurements.  One of the steps in the pipeline is the darken the fonts by making them a bit thicker.

Reply:
It looks like only the entries that had issues in `fontMetricsData.js` were updated.  Based on what I'm seeing in this diff things look fine.  If there's differences between the TTF and tfm file it's probably because of the darkening we do the font.